app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -310,6 +310,35 @@ public void setChatBackground() {
     }
 
     private void checkEnterKeyPref() {
+        mici=PrefManager.getBoolean(Constant.MIC_INPUT,true);
+        if(mici) {
+            btnSpeak.setImageResource(R.drawable.ic_mic_white_24dp);
+            btnSpeak.setOnClickListener(new View.OnClickListener() {
+
+                @Override
+                public void onClick(View v) {
+                    promptSpeechInput();
+                }
+            });
+        }
+        else {
+            btnSpeak.setImageResource(R.drawable.ic_send_fab);
+            btnSpeak.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    switch (v.getId()) {
+                        case R.id.btnSpeak:
+                            String message = etMessage.getText().toString();
```

Comment:
Same goes for here as well, etMessage makes no sense, so try to rename it to reflect its purpose in a better way.